lab_6_pipeline/pipeline.py

```diff
@@ -144,16 +255,57 @@ def __init__(self, corpus_manager: CorpusManager):
         """
         Initializes MorphologicalAnalysisPipeline
         """
+        self._corpus = corpus_manager
+        self._mystem = Mystem()
+        mapping_path = Path(__file__).parent / 'data' / 'mystem_tags_mapping.json'
+        self._converter = MystemTagConverter(mapping_path)
 
     def _process(self, text: str) -> List[ConlluSentence]:
         """
         Returns the text representation as the list of ConlluSentence
         """
+        conllu_sentences = []
+        sentences = split_by_sentence(text)
+        for position, sentence in enumerate(sentences):
+            mystem_sentence = self._mystem.analyze(sentence)
+            conllu_tokens = []
+            token_counter = 0
+            for token in mystem_sentence:
+                if not re.match(r'\w+|[.]', token['text']):
```

Comment:
what kind of tokens won't be processed?

Reply:
The ones that are not words/numbers/full stops